tests.py

```diff
@@ -41,38 +57,197 @@ def assertRegex(self, text, expected_regexp, msg=None):
                     raise self.failureException(msg)
 
 
+class GarbageCollectedMixin(object):
+    """A mixin for test cases that garbage collects before running."""
+
+    def setUp(self):
+        gc.collect()
+
+
 # Unit tests
 
+SINGLE_ELEMENT_OUTPUT = ('digraph ObjectGraph {\n'
+                         '  node[shape=box, style=filled, fillcolor=white];\n'
+                         '  ${label_a}[fontcolor=red];\n'
+                         '  ${label_a}[label="TestObject\\nTestObject(A)"];\n'
+                         '  ${label_a}[fillcolor="0,0,1"];\n'
+                         '}\n')
 
-def empty_edge_function(obj):
-    return []
+
+TWO_ELEMENT_OUTPUT = ('digraph ObjectGraph {\n'
+                      '  node[shape=box, style=filled, fillcolor=white];\n'
+                      '  ${label_a}[fontcolor=red];\n'
+                      '  ${label_a}[label="TestObject\\nTestObject(A)"];\n'
+                      '  ${label_a}[fillcolor="0,0,1"];\n'
+                      '  ${label_b} -> ${label_a};\n'
+                      '  ${label_b}[label="TestObject\\nTestObject(B)"];\n'
+                      '  ${label_b}[fillcolor="0,0,0.766667"];\n'
+                      '}\n')
```

Comment:
I think a tripple-quoted string, wrapped with `textwrap.dedent()`, would be more readable:
```python
TWO_ELEMENT_OUTPUT = textwrap.dedent('''\
    digraph ObjectGraph {
      ...
    }
''')
```
(the same applies to SINGLE_ELEMENT_OUTPUT as well)